src/target/riscv/batch.h

```diff
@@ -48,8 +159,10 @@ struct riscv_batch {
 	 * However, RISC-V DMI "busy" condition could still have occurred.
 	 */
 	bool was_run;
-	/* Idle count used on the last run. Only valid after `was_run` is set. */
-	size_t used_idle_count;
+	/* Number of RTI cycles used by the last scan on the last run.
+	 * Only valid when `was_run` is set.
+	 */
+	unsigned int used_delay;
```

Comment:
I'd recommend to name this "last_scan_delay" (or similar) to make its meaning more clear to the reader.

```suggestion
	unsigned int last_scan_delay;
```

Reply:
Addressed.